config.yml

```diff
@@ -83,8 +83,11 @@ params:
   assets:
     css:
       # Use https://www.srihash.org to generate the hash, or use the hash from upstream
-      uri:              "https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css"
-      sri:              "sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn"
+      uri:              "https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
```

Comment:
Might be better serving bootstrap from our server, without SRI hashes and everything.

We could later switch to npm.

Reply:
Switched to npm here :). I'm sure my implementation could be improved though, so happy to hear thoughts.